.github/workflows/pull_request_javascript_check.yml

```diff
@@ -16,4 +16,6 @@ jobs:
         node-version: 14.x
     - run: npm install
     - run: npm run lint
-    - run: npm test
\ No newline at end of file
+    - run: npm run mocha
+    # - run: npx playwright install-deps
+    # - run: npm test
```

Comment:
Unfortunately, `localhost:8080` is not available on the stock Github ubuntu build machine. :)